$DragonFly: src/usr.sbin/ntpd/patches/Attic/parse.y.patch,v 1.2 2004/11/09 21:29:34 joerg Exp $

--- parse.y.orig	2004-11-09 22:14:58.000000000 +0100
+++ parse.y	2004-11-09 22:15:12.000000000 +0100
@@ -39,7 +39,7 @@
 static FILE			*fin = NULL;
 static int			 lineno = 1;
 static int			 errors = 0;
-char				*infile;
+const char			*infile;
 
 int	 yyerror(const char *, ...);
 int	 yyparse(void);
@@ -476,7 +476,7 @@
 }
 
 int
-parse_config(char *filename, struct ntpd_conf *xconf)
+parse_config(const char *filename, struct ntpd_conf *xconf)
 {
 	struct sym		*sym, *next;
 
